api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -64,4 +64,23 @@ message RateLimit {
   // success.
   config.ratelimit.v3.RateLimitServiceConfig rate_limit_service = 7
       [(validate.rules).message = {required: true}];
+
+  // If set to true enables emitting X-RateLimit-* headers by the filter:
```

Comment:
Can you specify that this information needs to be returned from the RLS? I don't think we generate service docs right now (you can check) so it might not be possible to ref link to specific fields, but you could at least reference the proto using `:repo:` and then discuss different field names?